adafruit_veml6075.py

```diff
@@ -126,25 +132,25 @@ def _take_reading(self):
         #      (uva, uvb, uvcomp1, uvcomp2, dark))
 
     @property
-    def uva(self):
+    def uva(self) -> float:
         """The calibrated UVA reading, in 'counts' over the sample period"""
         self._take_reading()
         return self._uvacalc
 
     @property
-    def uvb(self):
+    def uvb(self) -> float:
         """The calibrated UVB reading, in 'counts' over the sample period"""
         self._take_reading()
         return self._uvbcalc
 
     @property
-    def uv_index(self):
+    def uv_index(self) -> float:
         """The calculated UV Index"""
         self._take_reading()
         return ((self._uvacalc * self._uvaresp) + (self._uvbcalc * self._uvbresp)) / 2
 
     @property
-    def integration_time(self):
+    def integration_time(self) -> Union[None, int]:
```

Comment:
This can't return `None`, so it should just be `int`

Reply:
oof.....why did I see that?